slash/exception_handling.py

```diff
@@ -83,7 +83,11 @@ def handling_exceptions(fake_traceback=True, **kwargs):
 
     if not PYPY and fake_traceback:
         # Only in CPython we're able to fake the original, full traceback
-        fake_tbs = create_traceback_proxy(frame_correction=2)
+        try:
+            fake_tbs = create_traceback_proxy(frame_correction=2)
+        except:
```

Comment:
@vmalloc I narrowed it down to only 2 types. Indeed better to narrow as much has possible.. I think these two shall be, probably,  enough. What do you say?